contents/data_vis_makie_layouts.md

```diff
@@ -166,6 +172,11 @@ Then, the `inset` is easily done, as in:
 ```jl
 @sco JDS.figure_box_inset()
 ```
+<!--
+At least change `right` of `inset_ax2` or add a protrusion so the xaxis label
+isn't cropped. Alternatively enlarge thr `right` value on purpose to make the
```

Comment:
that's exactly the point! in this scenario the inset plot extends more than needed it. It's kinda obvious, because is very visual.

Reply:
I'd say that "more than needed" is quite subjective ...  
And just that we are on the same page: On my screen the missing part of the inset is less then half a zero of the xticklabel 10. This is the case for the PDF as well as the website. And _less than half a zero_ at least for me is not "kinda obvious, because is very visual". That is why I suggest to move the inset "far" into the margin that minimum the whole "10" isn't visible.